requirements.txt

```diff
@@ -3,3 +3,4 @@ Django==3.0.4
 pandas==1.0.1
 psycopg2==2.8.4
 python-dotenv==0.12.0
+networkx==2.4
```

Comment:
We also need `pyyaml==5.3` for the yaml save/open implementation

Reply:
Switched back to JSON, so now no longer necessary? More discussion below...